src/facade/disk_connection_backpressure.h

```diff
@@ -0,0 +1,65 @@
+// Copyright 2025, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#pragma once
+
+#include <util/fibers/uring_file.h>
```

Comment:
you do not need uring or linux specific api for this, imho.

Reply:
There are 3 API's:

1. Uring (uring_file.h) -> asynchronous fiber blocking
2. Linux (file.h) -> `thread blocking` via writev/preadv
3. C++ files (fstream) - > `thread blocking`

Why would we block the entire thread on a disk write ?

Reply:
I meant to use `io/file.h` interface, you do not need to leak the implementation in h file. 
For epoll we can use `helio/util/fibers/fiber_file.h` though it's not a priority and it's fine to assume this feature is only for iouring enabled systems but in any case you will have to write code in such way that it will build on macos (so better not to leak iouring in header files)